docs/index.rst

```diff
@@ -12,32 +12,32 @@ Configuration
    :caption: Configuration
 
 
-* Add the Crawlera middleware including it into the ``DOWNLOADER_MIDDLEWARES`` in your ``settings.py`` file::
+* Add the Zyte Smart Proxy Manager middleware including it into the ``DOWNLOADER_MIDDLEWARES`` in your ``settings.py`` file::
 
     DOWNLOADER_MIDDLEWARES = {
         ...
-        'scrapy_crawlera.CrawleraMiddleware': 610
+        'scrapy_zyte_proxy.SmartProxyManagerMiddleware': 610
     }
 
 * Then there are two ways to enable it
 
   * Through ``settings.py``::
 
-      CRAWLERA_ENABLED = True
-      CRAWLERA_APIKEY = 'apikey'
+      ZYTE_SPM_ENABLED = True
+      ZYTE_SPM_APIKEY = 'apikey'
 
   * Through spider attributes::
 
       class MySpider:
-          crawlera_enabled = True
-          crawlera_apikey = 'apikey'
+          smp_enabled = True
+          smp_apikey = 'apikey'
 
 
-* (optional) If you are not using the default Crawlera proxy (``http://proxy.crawlera.com:8010``),
+* (optional) If you are not using the default Zyte Smart Proxy Manager proxy (``http://proxy.zyte.com:8011``),
```

Comment:
It is a good time to switch to secure proxy endpoint and RFC compliaint: `https://proxy.zyte.com:8014`

Reply:
Replaced.